src/main/java/org/candlepin/insights/inventory/InventoryService.java

```diff
@@ -73,6 +73,15 @@ protected CreateHostIn createHost(ConduitFacts conduitFacts, OffsetDateTime sync
         if (conduitFacts.getRhProd() != null) {
             rhsmFactMap.put("RH_PROD", conduitFacts.getRhProd());
         }
+        if (conduitFacts.getSysPurposeRole() != null && !conduitFacts.getSysPurposeRole().isEmpty()) {
+            rhsmFactMap.put("SYS_PURPOSE_ROLE", conduitFacts.getSysPurposeRole());
```

Comment:
Can we please rename to SYSPURPOSE_ROLE for consistency w/ RedHatInsights/rhsm-subscriptions#48 ?

Reply:
DONE